pkg/actors/actors.go

```diff
@@ -60,24 +61,25 @@ type Actors interface {
 }
 
 type actorsRuntime struct {
-	appChannel          channel.AppChannel
-	store               state.Store
-	placement           *internal.ActorPlacement
-	grpcConnectionFn    func(address, id string, namespace string, skipTLS, recreateIfExists, enableSSL bool) (*grpc.ClientConn, error)
-	config              Config
-	actorsTable         *sync.Map
-	activeTimers        *sync.Map
-	activeTimersLock    *sync.RWMutex
-	activeReminders     *sync.Map
-	remindersLock       *sync.RWMutex
-	activeRemindersLock *sync.RWMutex
-	reminders           map[string][]Reminder
-	evaluationLock      *sync.RWMutex
-	evaluationBusy      bool
-	evaluationChan      chan bool
-	appHealthy          bool
-	certChain           *dapr_credentials.CertChain
-	tracingSpec         config.TracingSpec
+	appChannel            channel.AppChannel
+	store                 state.Store
+	placement             *internal.ActorPlacement
+	grpcConnectionFn      func(address, id string, namespace string, skipTLS, recreateIfExists, enableSSL bool) (*grpc.ClientConn, error)
+	config                Config
+	actorsTable           *sync.Map
+	activeTimers          *sync.Map
+	activeTimersLock      *sync.RWMutex
+	activeReminders       *sync.Map
+	remindersLock         *sync.RWMutex
+	activeRemindersLock   *sync.RWMutex
+	reminders             map[string][]Reminder
+	remindersStateBackoff backoff.BackOff
```

Comment:
I am adding this backoff to retry writing to database in case of race condition.